src/commands/run.ts

```diff
@@ -50,6 +50,7 @@ export default async (functionName: string, cmd: { nondev: boolean; params: stri
  */
 async function runFunction(functionName: string, params: string[], scriptId: string, devMode: boolean) {
   try {
+    // TODO: @grant what are the issues/risks of not using local auth?
```

Comment:
I am not familiar with the run api. Does it requires a local auth and fails if global auth?